nav2_collision_monitor/src/pointcloud.cpp

```diff
@@ -97,7 +104,7 @@ void PointCloud::getParameters(std::string & source_topic)
     throw std::runtime_error{"Failed to lock node"};
   }
 
-  SourceBase::getParameters(source_topic);
+  getBasicParameters(source_topic);
```

Comment:
This name was introduced after merging `PolygonBase` with `Polygon` classes (https://github.com/ros-planning/navigation2/pull/2982#discussion_r892902866 thread). Since both classes have the same method `PolygonBase::getParameters()` and `Polygon::getParameters()` (first have common for all polygons params, seconds have only Polygon-specific ones), I've decided to rename `PolygonBase::getParameters() -> Polygon::getBasicParameters()` and remain `Polygon::getParameters() / Circle::getParameters()` for shape-specific params.

The same was made for `Source/Scan/Pointcloud` classes, but for naming unification.

Reply:
Got it - then its not basic as much as `common`, please rename to `getCommonParameters` :+1: I read that a completely different way with `basic`, that makes sense sorry for the confusion

Reply:
Done: https://github.com/ros-planning/navigation2/pull/2982/files#diff-d3d7f7e1c461b03d4a5868bd2766b722163399e0cb51227319a5b24e7cb8a434R149 and https://github.com/ros-planning/navigation2/pull/2982/files#diff-676640f2d50e736840827ec627347e72baace66bcbfff89d4c4020f389b27550R79